web/themes/custom/pineapple/components/_article-teaser-card.twig

```diff
@@ -0,0 +1,22 @@
+{% set type = type|default('article') %}
+{% set attributes = attributes.addClass([
```

Comment:
Attributes should not have any classes referencing nodes or any classes that come from Drupal. Any and all components shouldn’t have any references to Drupal

Reply:
That also means that this file should may be name _card.twig

Reply:
okay, done.